src-built-in/components/toolbar/stores/toolbarStore.js

```diff
@@ -246,33 +245,166 @@ class _ToolbarStore {
 	}
 
 	/**
-	 *
-	 *
-	 *
-	 */
+	* Load the hotkeys from the config.json. If there are no items in config.json, hotkeys use the defaults in the current file.
+  *
+  *
+	* @param {any} done
+	* @memberof _ToolbarStore
+	*/
+	setupHotkeys(done) {
+		// defaults for the hotkeys
+		const defaultHotkeys = [
```

Comment:
How about:
A) Moving the defaults array to a separate file that can be used in multiple places? 
B) Making the action strings constants.

Reply:
I like this idea. 
I think moving forward it would be a great idea as clashes in hotkeys are not ideal anyway!

Reply:
It's worth a conversation, I'm totally open to ideas on this one.

Reply:
I thought you were pulling the defaults from file. Is that not the case?

Reply:
Not defaults, just config overrides. The reason was not to break existing clients code. Thinking about it now I'm sure we could get rid of these. This would get rid of some of the other code below too. I need to revisit and reason to why I kept defaults.